scripts/drtprod

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+# drtprod is a wrapper for roachprod for use when managing the long-lived DRT
+# clusters that ensures consistent use of the correct project-assignment vars
+# and done some additional sanity check enforcement on some flags.
+
+if [[ $1 == "start" ]]; then
+  if [[ "$*" != *"--secure"* ]]; then
+    echo "--secure is required when starting DRT clusters"
+    exit 1
+  fi
+  if [[ "$*" != *"--sql-port 26257"* ]]; then
```

Comment:
Thanks for adding this, I added some additional checks in `roachprod` to prevent using DNS/services on clusters that do not support it.